### DISCUSSION TITLE
Justify Content seems not working for grid class ?

### DISCUSSION THREAD

**Discussion by tian-yan** [tailwindlabs/tailwindcss#4441] [Help] [answered]
From [official document](https://tailwindcss.com/docs/justify-content), it reads
> **Justify Content**
> Utilities for controlling how flex and grid items are positioned along a container's main axis.

I noticed that all the examples in that webpage are for `class="flex"`.  According to my below demo testing, **Justify Content** seemed not work regardless of whether I have `justify-items-center` in place or not. 
[Tailwind CSS Playground Demo here](https://play.tailwindcss.com/4KcMcI7AFT). 
So I am wondering if this **Justify Content** would work for `class="grid"` as per above official document's claiming ? 

If **Justify Content** was not for `class="grid"`, then do we have an equivalent for `class="grid"` to control the positions of grid items along the grid container's main axis ? something to produce same layout as shown in [this link](https://css-tricks.com/snippets/css/complete-guide-grid/#justify-content) ?
Thanks in advance for any inputs.

**Comment by simonswiss** [comment, marked as answer]
Hey!

In your example, that class will have no effect, since the grid cells are set to use all the available space (`1fr`). `justify-content` on a grid will only have an effect if the cells are not filling the entire available space.

I have updated your example with a custom `jit` class that sets hardcoded width to each grid columns, so that the grid doesn't use all the available space.

In this scenario, you will see the impact of changing `justify-end` to `justify-start`, etc.

https://play.tailwindcss.com/RdMh4O6Byt

Hope it helps!

**Reply by tian-yan** [reply]
Thanks for sharing, it worked perfectly. 
Activating "Just-in-Time Mode" in file "tailwind.config.js" enables the `grid-cols-[10%, 20%, 30%]`. Love it so much. :)
